cfme/tests/infrastructure/test_snapshot.py

```diff
@@ -100,6 +100,7 @@ def test_memory_checkbox(small_test_vm, provider, soft_assert):
 @pytest.mark.rhv1
 @pytest.mark.rhv3
 @test_requirements.rhev
+@test_requirements.snapshot
 @pytest.mark.meta(automates=[1571291, 1608475])
```

Comment:
@prichard77 
```test_requirements.snapshot``` is already declared globally. So we don't need to add it to each test case from this file.

Reply:
Removed